applications/nrf_desktop/doc/ble_bond.rst

```diff
@@ -40,9 +40,9 @@ Both IDs related to the given Bluetooth peer operation are propagated in ``ble_p
 
 The identity usage depends on the device type:
 
-* nRF Desktop central
+nRF Desktop central:
```

Comment:
I think you could still have the bullets.

Reply:
Please see the comment I left in reply to Greg's about this line.